src/Symfony/Bridge/Monolog/Processor/ProcessorInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Processor;
+
+/**
+ * An optional interface to allow autoconfiguration of Monolog processors.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface ProcessorInterface
```

Comment:
Funny, for twig you forced me to add this interface in twig itself.
Why didn't you do the same thing here?

Reply:
I did, see https://github.com/symfony/symfony/pull/28845
This has been reverted and added to Monolog meanwhile :)

Reply:
Ahah. Thanks